test/unit/specs/validators/alpha.spec.js

```diff
@@ -1,6 +1,10 @@
 import alpha from 'src/validators/alpha'
 
 describe('alpha validator', () => {
+  it('should validate undefined string', () => {
```

Comment:
I think that `should validate undefined` rather than `should validate undefined string` is appropriate.